Readme.md

```diff
@@ -7,28 +7,44 @@
 # Tracee: Runtime Security and Forensics using eBPF
 
 Tracee uses eBPF technology to tap into your system and give you access to hundreds of events that help you understand how your system behaves.
-In addition to basic observability events about system activity, Tracee adds a collection of sophisticated security events that expose more advanced behavioral patterns. You can also easily add your own events using the popular [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) language.
+In addition to basic observability events about system activity, Tracee adds a collection of sophisticated security events that expose more advanced behavioral patterns. 
 Tracee provides a rich filtering mechanism that allows you to eliminate noise and focus on specific workloads that matter most to you.
 
-To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee).
+**Key Features:**
+* Kubernetes native installation
+* Hundreds of default events
+* Ships with basic set of behavioral signatures for malicious detection out of the box 
+* Easy configuration through Tracee Policies 
+* Kubernetes native user experience that is targetted at cluster administrators
+
+> We release new features and changes on a monthly basis. Learn more about the letest release in our [discussions.](https://github.com/aquasecurity/tracee/discussions)
+
+To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee/latest/docs/overview/). 
 
 ## Quickstart
 
-You can easily start experimenting with Tracee using the Docker image as follows:
+Either:
```

Comment:
If I reading this in a fresh user mindset, I'm thinking "should I click into one of these tutorials or follow the steps below?"

Reply:
made changes to the subheadings before the bullet points